beeline/middleware/awslambda/__init__.py

```diff
@@ -148,4 +153,9 @@ def _beeline_wrapper(event, context):
             # we have to flush events before the lambda returns
             beeline.get_beeline().client.flush()
 
-    return _beeline_wrapper
+    def outer_wrapper(*args, **kwargs):
+        return beeline_wrapper(*args, record_input=record_input, record_output=record_output, **kwargs)
```

Comment:
Open to suggestions on a better name

Reply:
I think this is fine, it's internal and is clear enough :+1: